### PR TITLE
PYIC-3114 Added new cimit stub base proj. for cimit stub lambdas

### PR DESCRIPTION
**Pull request by rpayal**
## Proposed changes
Create initial getContraIndicators stub lambda

### What changed
- Added new cimit stub base proj. for cimit stub lambda's
- Added getContraIndicators lamdba with test.

### Issue tracking

- [PYI-3114](https://govukverify.atlassian.net/browse/PYI-3114)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: di-ipv-cimit-stub/gradle/wrapper/gradle-wrapper.jar
Comment (therealslimsladey) [inline di-ipv-cimit-stub/gradle/wrapper/gradle-wrapper.jar:1]: The gradle-wrapper.jar differs from the jar in the other stubs - please replace with the one from orchestrator-stub, for example.

File: di-ipv-cimit-stub/build.gradle [hunk 1]
```diff
@@ -0,0 +1,40 @@
+plugins {
+	id "java"
+	id "jacoco"
+}
+
+repositories {
+	mavenCentral()
+}
+
+sourceCompatibility = 17
```
Comment (therealslimsladey) [inline di-ipv-cimit-stub/build.gradle]: We're sticking with java 11 for the lambdas for now aren't we, also we should specify targetCompatibility = 11 as well

File: di-ipv-cimit-stub/lambdas/build.gradle [hunk 2]
```diff
@@ -0,0 +1,9 @@
+plugins {
+	id "java"
+}
+
+repositories {
+	mavenCentral()
+}
+
+sourceCompatibility = 17
```
Comment (therealslimsladey) [inline di-ipv-cimit-stub/lambdas/build.gradle]: source and target compatibility to 11

File: di-ipv-cimit-stub/lambdas/get-contra-indicators/build.gradle [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	id 'io.freefair.aspectj.post-compile-weaving' version '8.0.1'
+}
+
+group "uk.gov.di.ipv"
+
+repositories {
+	mavenCentral()
+}
+
+sourceCompatibility = 11
```
Comment (therealslimsladey) [inline di-ipv-cimit-stub/lambdas/get-contra-indicators/build.gradle:12]: and targetCompatibility to 11

File: di-ipv-cimit-stub/startup.sh
Comment (therealslimsladey) [inline di-ipv-cimit-stub/startup.sh:1]: copy/paste from one of the ecs lambdas? We don't need this do we?

**Review by therealslimsladey — CHANGES_REQUESTED** [review]
also seeing an Xlint:adviceDidNotMatch compilation warning - looking into that.

File: di-ipv-cimit-stub/gradle/wrapper/gradle-wrapper.jar
Comment (rpayal) [reply di-ipv-cimit-stub/gradle/wrapper/gradle-wrapper.jar:1]: checked shasum and updated file.

File: di-ipv-cimit-stub/lambdas/get-contra-indicators/build.gradle (hunk 3 above)
Comment (rpayal) [reply di-ipv-cimit-stub/lambdas/get-contra-indicators/build.gradle:12]: done

File: di-ipv-cimit-stub/lambdas/build.gradle (hunk 2 above)
Comment (rpayal) [reply di-ipv-cimit-stub/lambdas/build.gradle]: done

File: di-ipv-cimit-stub/build.gradle (hunk 1 above)
Comment (rpayal) [reply di-ipv-cimit-stub/build.gradle]: done

File: di-ipv-cimit-stub/startup.sh
Comment (rpayal) [reply di-ipv-cimit-stub/startup.sh:1]: yeah not needed for this module. Removed.

**Review by therealslimsladey — APPROVED** [review]

**Pull request merged**